mosviz/utils/cutout_tools.py

```diff
@@ -97,15 +109,32 @@ def make_cutouts(catalogname, imagename, image_label,
     cutcls = partial(Cutout2D, data, wcs=wcs, mode='partial')
 
     for position, x_pix, y_pix, row in zip(c, x, y, table):
-        try:
-            cutout = cutcls(position, size=(y_pix, x_pix))
-        except NoConvergence:
-            if verbose:
-                log.info('WCS solution did not converge: '
-                         'Skipping {0}'.format(row['id']))
-            continue
 
-        if np.array_equiv(cutout.data, 0):
+        # TODO: Better units handling in WCS.
+        if apply_rotation:
+            cutout_wcs = WCS(naxis=2)
+            cutout_wcs.wcs.ctype = ['RA---TAN', 'DEC--TAN']
+            cutout_wcs.wcs.crval = [position.ra.deg, position.dec.deg]
+            cutout_wcs.wcs.crpix = [(x_pix - 1) * 0.5, (y_pix - 1) * 0.5]
+            cutout_wcs.wcs.cdelt = [row['spatial_pixel_scale'].value] * 2
```

Comment:
Ah wait this is the issue - you are missing ``.to(u.degree)``. I think because this is missing, it's making the cutout cover a huge area, which means the whole image needs to be read in!